spatial_maps/stats.py

```diff
@@ -179,3 +182,67 @@ def prob_dist_1d(x, bins):
 
     H, _ = np.histogram(x, bins=bins, normed=False)
     return (H / len(x)).T
+
+
+def _mask_in_both_if_any_is_nan(c0, c1):
+    """
+    Mask values in both cubes, if it is nan in cube 0 or/and
```

Comment:
this is not only for cubes right? seems that it should work on n dimensional arrays

Reply:
dropped the whole function